api/swagger-spec/v1.json

```diff
@@ -4374,18 +4820,12 @@
       "consumes": [
        "*/*"
       ]
-     }
-    ]
-   },
-   {
-    "path": "/api/v1/watch/persistentvolumes",
-    "description": "API at /api/v1",
-    "operations": [
+     },
      {
-      "type": "json.WatchEvent",
-      "method": "GET",
-      "summary": "watch individual changes to a list of PersistentVolume",
-      "nickname": "watchNamespacedPersistentVolumeList",
+      "type": "unversioned.Status",
+      "method": "DELETE",
+      "summary": "delete collection of PersistentVolume",
+      "nickname": "deletecollectionNamespacedPersistentVolume",
```

Comment:
That's a good catch - although this is generated by our scripts.
This is some more generic issue - see e.g. that Get Node is also saying that node is namespaced:
https://github.com/kubernetes/kubernetes/blob/master/api/swagger-spec/v1.json#L2728

So this is unrelated to this PR.
I filed #18497 for it.